test/active_hashcash_test.rb

```diff
@@ -38,4 +39,15 @@ def test_hashcash_bits
     ActiveHashcash::Stamp.parse("1:20:220623:test::MPWRGuN3itbd1NiQ:008").update!(ip_address: "127.0.0.1")
     assert_equal(ActiveHashcash.bits + 3, controller.hashcash_bits)
   end
+
+  def test_hashcash_hidden_field_tag
```

Comment:
I'm not used to work wirh minitest. Please let me know if I should change something in the test